retrofit-adapters/rxjava2/pom.xml

```diff
@@ -23,6 +23,10 @@
       <groupId>io.reactivex.rxjava2</groupId>
       <artifactId>rxjava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.reactivestreams</groupId>
+      <artifactId>reactive-streams</artifactId>
+    </dependency>
```

Comment:
I’m not quite sure why this is necessary but both IntelliJ and `mvn verify` were unhappy without it.